controllers/drcluster_controller.go

```diff
@@ -264,17 +278,32 @@ func (u *drclusterUpdater) statusConditionSet(
 	return nil
 }
 
-func (u *drclusterUpdater) statusUpdate() error {
-	return u.client.Status().Update(u.ctx, u.object)
+func (u *drclusterInstance) statusUpdate() error {
+	if !reflect.DeepEqual(u.savedInstanceStatus, u.object.Status) {
+		if err := u.client.Status().Update(u.ctx, u.object); err != nil {
+			u.log.Info(fmt.Sprintf("Failed to update drCluster status (%s/%s/%v)",
```

Comment:
(nit) remove duplicate formated string
```suggestion
                      msg = fmt.Sprintf("Failed to update drCluster status (%s/%s/%v), u.object.Name, u.object.Namespace, err)
			u.log.Info(msg)
			return fmt.Errorf(msg)
```